numpy/distutils/system_info.py

```diff
@@ -1030,20 +1027,7 @@ def calc_info(self):
 
 
 class lapack_mkl_info(mkl_info):
-
```

Comment:
I left it empty for consistency with the other mkl_info subclass blas_mkl_info, which was also empty. I guess you could remove both and just use the parent mkl_info class for both blas_mkl and lapack_mkl in the cl dictionary.